### ISSUE TITLE
Suggestion for: <string name="privacyText">

### ISSUE THREAD

**Issue by DrJonesKlaus** [thunderbiscuit/padawan-wallet#329]
What do you think about including the Twitter/X username and the email so "lazy" people can send comments easier as follows?

<string name="privacyText">Padawan Wallet does not collect any information on its users, and does not use Firebase Crashlytics. This, however, means it\'s much harder for us to hear about the bugs from our users. If you have feedback, feel free to send a message to the Padawan Twitter account **(@padawanwallet)** or to write us an email **(padawanemail@email.com)**. You can read our full privacy policy here:</string>

**Comment by thunderbiscuit** [comment]
Yep that's a good little addition. Unfortunately I don't have an official padawan email but I could look into getting that going maybe.
